src/content/en/2021/cdn.md

```diff
@@ -1,202 +1,198 @@
 
-**Single geographic location:** Tests are run from a single datacenter and cannot test the geographic distribution of many CDN vendors.
+- **Single geographic location:** Tests are run from a single datacenter and cannot test the geographic distribution of many CDN vendors.
 
-**Cache effectiveness:** Each CDN uses proprietary technology and many, for security reasons, do not expose cache performance.
+- **Cache effectiveness:** Each CDN uses proprietary technology and many, for security reasons, do not expose cache performance.
 
-**Localization and internationalization:** Just like geographic distribution, the effects of language and geo-specific domains are also opaque to these tests.
+- **Localization and internationalization:** Just like geographic distribution, the effects of language and geo-specific domains are also opaque to these tests.
 
-**CDN detection:** This is primarily done through DNS resolution and HTTP headers. Most CDNs use a DNS CNAME to map a user to an optimal datacenter. However, some CDNs use AnyCast IPs or direct A+AAAA responses from a delegated domain which hide the DNS chain. In other cases, websites use multiple CDNs to balance between vendors, which is hidden from the single-request pass of our crawler.
+- **CDN detection:** This is primarily done through DNS resolution and HTTP headers. Most CDNs use a DNS CNAME to map a user to an optimal datacenter. However, some CDNs use Anycast IPs or direct A+AAAA responses from a delegated domain which hide the DNS chain. In other cases, websites use multiple CDNs to balance between vendors, which is hidden from the single-request pass of our crawler.
 
-All of this limits the effectiveness in the measurements.
+All of this influences our measurements.
 
-Most importantly, these results reflect the utilization of specific features (Example: TLS, HTTP/2 etc.,) but do not reflect actual impact. YouTube is more popular than "www.example.com" yet both will appear as equal value when comparing utilization.
+Most importantly, these results reflect the utilization of specific features (Example: TLS, HTTP/2 etc.,) per site, but do not reflect actual traffic usage. YouTube is more popular than "www.example.com" yet both will appear as equal value when comparing utilization.
 
 With this in mind, here are a few statistics that were intentionally not measured in the context of a CDN:
 
-1. Time To First Byte(TTFB)
+1. Time To First Byte (TTFB)
 2. CDN Round Trip Time
 3. Core Web Vitals
-4. Cache Hit v/s Cache Miss performance etc.,
+4. Cache Hit versus Cache Miss performance etc.
+
+While some of these could be measured with our crawl, and others by using the CrUX dataset, the limitations of our crawl and the use of multiple CDNs by some sites, will be difficult to measure and so could be incorrectly reported against the CDN used in out crawl. For these reasons, we have decided not to measure these statistics in this chapter.
```

Comment:
I added this sentence to explain why (I think) you choose not to measure these. Please review and let me know if you want to change.

Reply:
Yes, the addition makes sense. Adds more color to the statement.

Reply:
Just spotted a typo and repeated use of the phrase “our crawl” so let’s use below tweak instead:

```suggestion
While some of these could be measured with HTTP Archive dataset, and others by using the CrUX dataset, the limitations of our methodology and the use of multiple CDNs by some sites, will be difficult to measure and so could be incorrectly attributed. For these reasons, we have decided not to measure these statistics in this chapter.
```